src/components/MediaSettings/VideoBackgroundEditor.vue

```diff
@@ -36,43 +39,76 @@
 			<Blur :size="20" />
 			{{ t('spreed', 'Blur') }}
 		</button>
-		<!-- hide custom background for now -->
-		<button v-if="false" key="upload" class="background-editor__element">
-			<ImagePlus :size="20" />
-			{{ t('spreed', 'Custom') }}
-		</button>
-		<button v-for="path in backgrounds"
-			:key="path"
-			aria-label="TODO: add image names as aria labels"
-			class="background-editor__element"
-			:class="{'background-editor__element--selected': selectedBackground === path}"
-			:style="{
-				'background-image': 'url(' + path + ')'
-			}"
-			@click="handleSelectBackground(path)">
-			<CheckBold v-if="selectedBackground === path"
-				:size="40"
-				fill-color="#fff" />
-		</button>
+		<template v-if="hasBackgroundsCapability">
```

Comment:
```suggestion
		<template v-if="predefinedBackgrounds?.length">
```
Same approach as for supported reactions, without dedicated computed property... (see below)